code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -852,30 +852,37 @@
 			fail_msg = "There is no exposed flesh or thin material [target_zone == "head" ? "on [p_their()] head" : "on [p_their()] body"] to inject into."
 		to_chat(user, "<span class='alert'>[fail_msg]</span>")
 
+///
+/**
+ * Gets the obscured ITEM_SLOTs on a human
+ *
+ * Returns: 
+ * * A bitfield containing the ITEM_SLOTS bitflags that are obscured.
+ */
 /mob/living/carbon/human/proc/check_obscured_slots()
-	var/list/obscured = list()
+	var/obscured // CHAP-TODO: Make this a bitfield maybe?
 
 	if(wear_suit)
 		if(wear_suit.flags_inv & HIDEGLOVES)
-			obscured |= SLOT_HUD_GLOVES
+			obscured |= ITEM_SLOT_GLOVES
 		if(wear_suit.flags_inv & HIDEJUMPSUIT)
-			obscured |= SLOT_HUD_JUMPSUIT
+			obscured |= ITEM_SLOT_ICLOTHING
 		if(wear_suit.flags_inv & HIDESHOES)
-			obscured |= SLOT_HUD_SHOES
+			obscured |= ITEM_SLOT_SHOES
 
 	if(head)
 		if(head.flags_inv & HIDEMASK)
-			obscured |= SLOT_HUD_WEAR_MASK
+			obscured |= ITEM_SLOT_MASK
 		if(head.flags_inv & HIDEEYES)
-			obscured |= SLOT_HUD_GLASSES
+			obscured |= ITEM_SLOT_EYES
 		if(head.flags_inv & HIDEEARS)
-			obscured |= SLOT_HUD_RIGHT_EAR
-			obscured |= SLOT_HUD_LEFT_EAR
+			obscured |= ITEM_SLOT_RIGHT_EAR
+			obscured |= ITEM_SLOT_LEFT_EAR
 
-	if(length(obscured) > 0)
+	if(obscured)
 		return obscured
 	else
-		return null
+		return NONE
```

Comment:
reduce this to just `return obscured` with `var/obscured = NONE` at the top please